profile/scop/team.md

```diff
@@ -9,38 +9,41 @@ Our team today is composed of more than 60 people:
 | Samuel Bigard           | developer                                  | Lille       | @samuelbigard       |
-| Alexis Bouchez          | developer                                  | Lille       | @alexisbouchez      |
-| Justine Bruneau Zahiri  | frontend developer                         | Paris       | @justinezahiri      |
+| Thomas Billard          | backend developer                          | Paris       | @thbil              |
 | Antoine Bluchet         | backend developer                          | Nantes      | @soyuka             |
 | Pascal Borreli          | consultant                                 | Paris       | @pborreli           |
+| Justine Bruneau-Zahiri  | frontend developer                         | Paris       | @justinezahiri      |
 | Antoine Cau             | developer                                  | Lille       | @AntoineCAU         |
 | Vincent Chalamon        | lead developer                             | Lille       | @vincentchalamon    |
 | Robin Chalas            | consultant                                 | Lyon        | @chalasr            |
 | Thomas Colin            | frontend developer                         | Nantes      | @thcolin            |
 | Grégory Copin           | technical director                         | Lille       | @Gregcop1           |
-| Olivia De Scheerder     | developer                                  | Nantes      | @Palingenae         |
+| Valentin Crochemore     | frontend developer                         | Paris       |                     |
 | Justine Delalleau       | frontend developer                         | Lille       | @chuticat           |
-| Julie Ducourneau        | frontend developer                         | Nantes      | @JulieNobody        |
+| Olivia De-Scheerder     | developer                                  | Nantes      | @Palingenae         |
+| Julie Ducourneau        | backend developer                          | Nantes      |                     |
 | Kévin Dunglas           | manager and technical director             | Lille       | @dunglas            |
 | Jonas Duval             | backend developer                          | Paris       | @jovuk              |
 | Élise Duverdier         | backend developer                          | Paris       | @eliseduverdier     |
-| Camille Gaquière        | project manager                            | Rennes      |                     |
-| Saif Eddin Gmati        | backend developer                          | Tunis       | @azjezz             |
 | Margaux Feslard         | backend developer                          | Lille       | @Margauxfeslard     |
 | Julien Francoz          | SRE                                        | Nantes      | @jfcoz              |
-| Arnaud Frézet           | backend developer                          | Lyon        | @johnkrovitch       |
+| Arnaud Frezét           | backend developer                          | Lyon        | @johnkrovitch       |
```

Comment:
To revert.

Reply:
Bien vu, je suis partie de la liste du groupe Google, il y a une faute là alors 😇